popupwindow.js

```diff
@@ -0,0 +1,121 @@
+const exitPopover = document.getElementById("close-popover");
+
+btn_button.addEventListener("click", () => {
+  div.classList.add("popover-display");
+});
+mobileBtn.addEventListener("click", () => {
+  nav.classList.add("menu-btn");
+});
+
+function display() {
+  nav.classList.remove("menu-btn");
+}
+
+exitBtn.addEventListener("click", display);
+aboutBtn.addEventListener("click", display);
+contactBtn.addEventListener("click", display);
+
+function popover() {
+  div.classList.remove("popover-display");
+}
+exitPopover.addEventListener("click", popover);
+
+const dataPopover = [
+  {
+    heading: "Tonic",
+    image: "./imagsPop/tonic.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/multi1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Facebook",
+    image: "./imagsPop/facebook1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/project image 2.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+];
+
+const modal = document.querySelector(".modal");
+const popupWrap = document.querySelector(".pop-up-wrap");
+const projectBtn = document.querySelectorAll(".see-btn");
+
+projectBtn.forEach(function (Btn) {
+  Btn.addEventListener("click", function (e) {
+    const i = e.currentTarget.dataset.id;
+    popupWrap.innerHTML = `<div class="pop-up-nav">
+		  <h2 class="poppins" id="logo">${dataPopover[i].heading}</h2>
+		  <span class = "pop-up-closeBtn">&times;</span>
```

Comment:
- [ ]  it would be better if you use the same button in the Figma design in order to match the design 
- Yours 
<img width="48" alt="Screen Shot 2022-10-19 at 18 04 45" src="https://user-images.githubusercontent.com/61790587/196757775-5175188d-77a2-421e-9022-2ee0e63bb6dc.png">
- Figma 
<img width="49" alt="Screen Shot 2022-10-19 at 18 04 50" src="https://user-images.githubusercontent.com/61790587/196757786-2cfdb7eb-b64e-41eb-b5ad-82d471724672.png">

Reply:
I have gotten this Icon from Figma design please look closely.